src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig

```diff
@@ -25,6 +25,13 @@
             <b>Macro Calls</b>
             <span class="sf-toolbar-status">{{ collector.macrocount }}</span>
         </div>
+        <div class="sf-toolbar-info-piece">
+            <input type="color" id="__twig-debug-color" value="#ff0000">
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <span><a href="javascript:;" id="__twig-debug-enable">Enable debug</a></span>&nbsp;
```

Comment:
`href="javascript:;"` breaks the CSP compatibility. Use button tags instead (which are semantically correct) or `href="#"` with `event.preventDefault()` in the listener (more complex due to browser compat as this is part of the toolbar, not of the profiler UI for which we can  afford supporting only modern browsers).